core/model/modx/transport/modtransportpackage.class.php

```diff
@@ -216,9 +216,9 @@ public function getTransport($state = -1) {
                     }
                     if ($transferred) {
                         if ($state < 0) {
-                            /* if directory is missing but zip exists, and DB state value is incorrect, fix here */
+                            /* if directory is missing or empty but zip exists, and DB state value is incorrect, fix here */
                             $targetDir = basename($sourceFile, '.transport.zip');
-                            $state = is_dir($packageDir.$targetDir) ? $this->get('state') : xPDOTransport::STATE_PACKED;
+                            $state = (is_dir($packageDir.$targetDir) && count(glob($packageDir.$targetDir.'/*')) !== 0) ? $this->get('state') : xPDOTransport::STATE_PACKED;
```

Comment:
Some whitespace is missing between the dots.

Reply:
@Jako you mean here `$packageDir.$targetDir`? E.g. should be `$packageDir . $targetDir` etc. ?